### ISSUE TITLE
Hungarian queue has stopped working

### ISSUE THREAD

**Issue by meskobalazs** [mozillabrasil/sumo_live_helper#198]
### Description

I'm no longer receiving notifications for the Hungarian queue. English still works perfectly fine.

### Steps to Reproduce

1. Open the settings
2. Select all products and Hungarian for language
3. Refresh

**Expected behavior:**

I should receive notifications for the new threads.

**Actual behavior:**

There are no notifications for the queue.

### System information

* Operating system and Firefox version: Fedora 39 + Firefox 123.0 / Ubuntu 23.10 + Firefox 123.0 / Windows 10 + Firefox 25.0b3 (Developer Edition)

**Comment by dannycolin** [comment]
> I believe could be a good addition, what do you think

I'm fine with offering this option if someone wants to contribute a patch.

**Comment by jhonatasrm** [comment]
Yes, it looks good to me. I think a good addition would be an option to filter for open questions (not set as resolved) with or without interactions from the community.

**Comment by meskobalazs** [comment]
What I find puzzling is that this request yields results currently: <https://support.mozilla.org/api/2/question/?format=json&ordering=-id&is_solved=false&is_spam=false&is_locked=false&is_archived=false&locale=hu&num_answers=0>. 

Even filtering for product is working. But still, the extension does not show anything.

**Comment by jhonatasrm** [comment]
Currently, we notice that when we send the product name using the Upper Camel Case pattern (Firefox, Thunderbird...), it impacts the display of the expected result, constituting a bug. We must also analyze the handling of other product names in this regard.

**We need to pass the product name using Lower Camel Case (firefox, thunderbird...).**

Current querying by product:

 Firefox: [https://support.mozilla.org/api/2/question/?format=json&ordering=-id&is_solved=false&is_spam=false&is_locked=false&product=Firefox&is_taken=false&is_archived=false&locale=hu&num_answers=0](https://support.mozilla.org/api/2/question/?format=json&ordering=-id&is_solved=false&is_spam=false&is_locked=false&product=Firefox&is_taken=false&is_archived=false&locale=hu&num_answers=0)
Thunderbird: [https://support.mozilla.org/api/2/question/?format=json&ordering=-id&is_solved=false&is_spam=false&is_locked=false&product=Thunderbird&is_taken=false&is_archived=false&locale=hu&num_answers=0](https://support.mozilla.org/api/2/question/?format=json&ordering=-id&is_solved=false&is_spam=false&is_locked=false&product=Thunderbird&is_taken=false&is_archived=false&locale=hu&num_answers=0)

Desired querying:

Firefox: [https://support.mozilla.org/api/2/question/?format=json&ordering=-id&is_solved=false&is_spam=false&is_locked=false&product=firefox&is_taken=false&is_archived=false&locale=hu&num_answers=0](https://support.mozilla.org/api/2/question/?format=json&ordering=-id&is_solved=false&is_spam=false&is_locked=false&product=firefox&is_taken=false&is_archived=false&locale=hu&num_answers=0)
Thunderbird: [https://support.mozilla.org/api/2/question/?format=json&ordering=-id&is_solved=false&is_spam=false&is_locked=false&product=thunderbird&is_taken=false&is_archived=false&locale=hu&num_answers=0](https://support.mozilla.org/api/2/question/?format=json&ordering=-id&is_solved=false&is_spam=false&is_locked=false&product=thunderbird&is_taken=false&is_archived=false&locale=hu&num_answers=0)

Additionally, we have another function that checks the creation time of the question to determine if it should be displayed within a certain deadline.

```
function isWithinTimeRange(timeString) {
    const timeLimit = 24;     // In hours
    let currentTime = new Date();
    const timezone = currentTime.getTimezoneOffset() * 60 * 1000;     // Must manually remove the timezone offset
    currentTime = new Date(currentTime - timezone);                   // because SUMO API displays a local timestamp, but labels it UTC
    const minimumTime = currentTime.getTime() - (timeLimit * 3600000);
    const questionTime = new Date(timeString).getTime();

    return questionTime >= minimumTime;
}
```

**Comment by jhonatasrm** [comment]
I'll be addressing this bug. Additionally, I've created an enhancement issue directly linked to the bug of this filter for Hungarian. It is:
- #200 (related to the function I mentioned earlier)
